simapp/app.go

```diff
@@ -536,7 +543,14 @@ func (app *SimApp) setAnteHandler(txConfig client.TxConfig) {
 		panic(err)
 	}
 
-	app.SetAnteHandler(anteHandler)
+	// Create the CircuitBreakerAnteHandler with the CircuitKeeper
+	cbAnteHandler := ante.NewCircuitBreakerAnteHandler(cbKeeper, defaultAnteHandler)
+
+	// Chain the default AnteHandlers with the CircuitBreakerAnteHandler
+	allAnteHandlers := app.chainAnteDecorators(cbAnteHandler)
+
+	// Set the AnteHandler for the app
+	app.SetAnteHandler(allAnteHandlers)
```

Comment:
i dont believe this works, we will either need to append it to the list or create the antehandler in simapp